ewcfg.py

```diff
@@ -4263,6 +4308,14 @@ def wef_tool(ctn = None):
 		str_desc = "This is The Bassed God. He's gonna fuck your bitch.",
 		slime = fish_slime_saltwater
 	),
+        EwFish(
+		id_fish = "dirtybass",
+		str_name = "Dirty Bass",
+		rarity = fish_rarity_rare,
+		catch_time = None,
+		catch_weather = None,
+		str_desc = "A Dirty Bass Guitar. Maybe you should try cleaning it?",
+	),
```

Comment:
sounds like it should be an item rather than a fish

Reply:
Bass comes from the sea